src/__tests__/lib/utils/SynapseClient.test.tsx

```diff
@@ -17,8 +17,8 @@ describe('it works at integration level testing', () => {
       BackendDestinationEnum.REPO_ENDPOINT,
     ).catch((error) => {
       expect(error.status).toEqual(404)
-      expect(error.reason).toEqual(
-        'GET was not found. Please reference API documentation at https://docs.synapse.org/rest/',
+      expect(error.reason).toContain(
+        'GET was not found.',
```

Comment:
Nice--do you know if there is a PLFM issue to change these links in the backend?

Reply:
Good point, opened PLFM-6740